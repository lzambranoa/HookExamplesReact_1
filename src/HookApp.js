import React from 'react';

export const HookApp = () => {
    return (
        <div>
            <h1>Hola Leonardo!!!!</h1>
        </div>
    )
}