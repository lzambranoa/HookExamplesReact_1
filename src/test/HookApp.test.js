import React from 'react';

import { HookApp } from '../HookApp';
import { shallow } from 'enzyme';


describe('Pruebas de <HookApp />', () => {

    test('Debe mostrar correctamente el componente', () => {


        const wrapper = shallow( <HookApp />);
        expect( wrapper ).toMatchSnapshot();
    })
})