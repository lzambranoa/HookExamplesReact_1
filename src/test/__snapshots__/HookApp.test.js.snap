// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas de <HookApp /> Debe mostrar correctamente el componente 1`] = `
<div>
  <h1>
    Hola Leonardo!!!!
  </h1>
</div>
`;
